### DISCUSSION TITLE
unable to connect my wallet

### DISCUSSION THREAD

**Discussion by Not-Sarthak** [Cyfrin/foundry-full-course-cu#778] [Q&A]
can anyone help me with this? 
I am unable to connect my wallet. thanks :)

https://github.com/Cyfrin/foundry-full-course-f23/assets/92942966/22ffb81f-00d4-4f4d-9807-154aba2dff70

**Comment by theirrationalone** [comment]
Please Login to `Metamask` first.... due to timeout metamask might got locked. You'll have to enter your password again.
After doing this, whenever you'll try to connect with metamask mask again, A request for connection will popup on your metamask.

**Reply by Not-Sarthak** [reply]
I tried this but it is still not working. I even tried reinstalling wallets and even shifted to a different browser but it is not working anywhere for me

**Reply by theirrationalone** [reply]
Try to mitigate with this [comment](https://github.com/Cyfrin/foundry-full-course-f23/discussions/706#discussioncomment-6815363)

**Reply by Not-Sarthak** [reply]
so sorry to bother you again, but the issue still persists :(

**Reply by theirrationalone** [reply]
One more mitigation we can do is.... try to create a new sepolia rpc-url (alchemy) set that to metamask and then try again..

If even this mitigation doesn't work.... There are more chances you're facing `CORS` issue or `ALCHEMY_THROUGHPUT` Reach issue. (Maybe) 😅

**Reply by Not-Sarthak** [reply]
so i tried adding a new sepolia rpc url but it did not solve the issue, causing the same error again. Apart from this i've no idea about the `ALCHEMY_THROUGHPUT` reach issue, will have to look into it. 
thanks for taking out this much of time for helping me out btw :))
